### ISSUE TITLE
Can't control fan speeds properly on my 7800XT, AMD Adrenalin seems to take over

### ISSUE THREAD

**Issue by AnonymousWP** [Rem0o/FanControl.Releases#2652]
**Describe the bug**

I notice that when I have setup fan controls for my 7800XT, Adrenalin seems to adjust them. How do I know? Well, the fan curves in Adrenalin seem much different than what I've set. Relevant screenshots:

![image](https://github.com/user-attachments/assets/7e09d2a9-e71b-43a0-9769-84d93ce6dbd7)

![image](https://github.com/user-attachments/assets/9977e216-3c73-452a-9b0e-fdd1a89c2e24)

![image](https://github.com/user-attachments/assets/796d0a14-5d0b-488a-83a9-0b3d01773972)

Also if I change settings in Adrenalin, not much later (after something in FanControl gets triggered), Adrenalin seems to modify it.

**Is there a log.txt file next to FanControl.exe with recent date entries?**

Nope.

**Relevant hardware specs and setup**

[XFX Speedster QICK 319 Radeon RX 7800 XT Core Edition](https://tweakers.net/pricewatch/1976492/xfx-speedster-qick-319-radeon-rx-7800-xt-core-edition.html)

**Comment by Rem0o** [comment]
You should not look at what adrenaline shows you for the curve as it resets when 0 RPM kicks in. The curve fan control uses is not transferred to adrenaline.

**Comment by Rem0o** [comment]
From your configuration with the trigger, your fans would be 0 RPM until 80 deg, which is way to high. Cards usually permit up in the 45-60 deg range.  Just look at the graph you sent in adrenaline, the cutoff on the left is your 0 RPM cutoff temperature permitted by your card. You need to play around that limitation.

**Comment by AnonymousWP** [comment]
> The curve fan control uses is not transferred to adrenaline.

Is this technically doable by the way? Any chance we can see being supported in the future or does it depend on ADLX?

> From your configuration with the trigger, your fans would be 0 RPM until 80 deg, which is way to high.

Yeah fair, it was for testing purposes.

> Just look at the graph you sent in adrenaline, the cutoff on the left is your 0 RPM cutoff temperature permitted by your card. You need to play around that limitation.

Thanks. From my understanding and current screenshot, it seems that 60 degrees is safe

![image](https://github.com/user-attachments/assets/36317134-a41b-4ede-8791-e3784ff8933d)

> I would agree I should write some documentations around GPU setup, as they imply many unclear constraints as to how you can set them up from a third party.

I'm thinking about writing some documentation to help out, because I really like this application and because I initially got confused as there were no clear steps about how setup FanControl for an AMD GPU (which options you need to check, what the procedures are in Adrenalin etc.). Where would you want me to put that documentation? Wiki? Website? Or are you okay with the README? First two options I obviously can't do; Wiki is only writable by maintainers. README I can do.

**Comment by cosmic-banana** [comment]
> You should not look at what adrenaline shows you for the curve as it resets when 0 RPM kicks in.

What do you mean by this? Is the Adrenaline feature for fan tuning not working at all?

I'm writing in this bug report because it seems bang-on the same issue I'm experiencing, yet the discussion so far has not touched on the relevant area for me, which relates to original poster's comment: "Also if I change settings in Adrenalin, not much later (after something in FanControl gets triggered), Adrenalin seems to modify it."

I do not have any curves for my GPU fan in Fan Control. In fact, I tried removing the pairing entirely hoping I would avoid this issue, but it persists. Fan Control, which is awesome software that I **want** to use, is interfering badly with Adrenaline, which is garbage software that I **have** to use.

Simply opening or closing the Fan Control application (or even letting it run for a while) seems to cause Adrenaline to reset/alter settings under Performance --> Tuning tab. This would be bad enough if it was only messing up the GPU fans, but the Tuning tab also configures clock speed and voltage.

I am a new user of both Fan Control and AMD Adrenaline and I've never had a working setup.

Because of this software conflict I am unable to use Fan Control :(

**Comment by Rem0o** [comment]
You can simply not enable your gpu fan from fancontrol or even completely disable ADLX all together.

**Comment by AnonymousWP** [comment]
But now him and me still don't have answers on our questions. 🤔

**Comment by Rem0o** [comment]
> Is this technically doable by the way? Any chance we can see being supported in the future or does it depend on ADLX?

Yes, the API supports changing the build-in graph, but it wouldn't work with how FanControl operates. 

> Where would you want me to put that documentation? Wiki? Website? 

The website /docs would be the way to go.  It's open-source you could open a PR, but I will most likely do it myself soon.

**Comment by AnonymousWP** [comment]
> > Where would you want me to put that documentation? Wiki? Website?
> 
> The website /docs would be the way to go. It's open-source you could open a PR, but I will most likely do it myself soon.

I'll wait for you to do it then. If you did it and I think it can be improved, I'll open a PR. 🙂

**Comment by Rem0o** [comment]
@AnonymousWP  Gave it a first shot with most often given answers:
https://getfancontrol.com/docs/

**Comment by cosmic-banana** [comment]
> You can simply not enable your gpu fan from fancontrol or even completely disable ADLX all together.

I don't know what you mean by 'not enable your gpu fan', since I never assigned a curve to it and didn't manually control the fan speed. I also tried removing the gpu fan pairing so Fan Control wasn't aware of it at all. It didn't fix the problem.

BUT, disabling ADLX as you suggested has fixed the issue I was experiencing! Now I can use Fan Control and Adrenaline at the same time! :)

**Comment by AnonymousWP** [comment]
> @AnonymousWP Gave it a first shot with most often given answers: https://getfancontrol.com/docs/

Looks good on the first glance, and I like the caveats you've put. However, I expected more of a step-by-step 'guide' though, like

> 
> ## Setup FanControl for AMD cards
> 
> 1. Check this
> 2. Do that

Unfortunately documentation is tsx format, which I'm not familiar with at all, so unfortunately I can't contribute.😕 Let me know if you'll still work on that. If not, you can close this issue.

**Comment by HTVX01** [comment]
I had the same problem. After installing the drivers, fan control did not find the gpu (sensor and card). After a whole day of searching for the problem, I found that the "full install" version of radeon adrenaline software blocks this option. When installing "drivers only / minimal install" everything works fine

![Image](https://github.com/user-attachments/assets/0fa6be8e-6674-4036-87aa-385f7dca03a6)

**Comment by imjkungz** [comment]
Hi, 
I have the same problem, please try to disable Zero RPM .

![Image](https://github.com/user-attachments/assets/f6b6d888-170a-4cc5-889c-3f2a13cf0b35)

**Comment by Gl6V7QvDIRH8gKPMItFtgVXa** [comment]
It is still an issue for me as well with 6800 XT. With full Adrenaline install it takes over the control and I have to  switch the tuning profile to Default and then the sensor is detected in Fan Control.
